File: frontend/src/scripts/realisticbiomes/Yield.ts
import Item from "../minecraft/Item.js";

export default class Yields {
    yields: Yield[]

    constructor(yields: Yield[]) {
        this.yields = yields
    }
}

export class Yield {
    item: Item
    amount: number

    constructor(item: Item, amount: number) {
        this.item = item
        this.amount = amount
    }
}

File: converter/src/main/java/net/stzups/civref/converter/realisticbiomes/Yield.java
package net.stzups.civref.converter.realisticbiomes;

import net.stzups.civref.converter.minecraft.Item;

class Yield {
    private final Item item;
    private final int amount;

    Yield(Item item, int amount) {
        this.item = item;
        this.amount = amount;
    }
}
